src/components/taskDetails/index.tsx

```diff
@@ -39,15 +35,15 @@ function Button(props: ButtonProps) {
         </button>
     );
 }
-function Textarea(props: TextAreaProps) {
-    const { name, value, onChange } = props;
+export function Textarea(props: TextAreaProps) {
```

Comment:
Why `testId` as a separate required prop?

Reply:
This component is only being used here and It's not reused in any other component, So I am passing it as a prop